build.cmd

```diff
@@ -768,7 +768,7 @@ if %__BuildNativeCoreLib% EQU 1 (
         set COMPlus_PartialNgen=1
     )
 
-    set NEXTCMD="%__CrossgenExe%" %__IbcTuning% /Platform_Assemblies_Paths "%__BinDir%"\IL /out "%__BinDir%\System.Private.CoreLib.dll" "%__BinDir%\IL\System.Private.CoreLib.dll"
+    set NEXTCMD="%__CrossgenExe%" /Platform_Assemblies_Paths "%__BinDir%"\IL /out "%__BinDir%\System.Private.CoreLib.dll" "%__BinDir%\IL\System.Private.CoreLib.dll"
```

Comment:
Why this change? We still want to optimize the image using IBC data if they are available. 

To disable partial NGen, I think we should rather delete setting of `COMPlus_PartialNGen`.

Reply:
Agreed with @jkotas on this.

Reply:
You should probably also remove references to ```__PartialNgen```.

Reply:
I was under the impression that if we do not provide IBC tuning data, we won't compile a partial NGen image. I was not aware that we could use IBC data for full NGen images too. Thanks for clarifying.